src/Struct/FindologicProduct.php

```diff
@@ -856,7 +856,7 @@ protected function setCategoriesAndCatUrls(): void
             $this->parseCategoryAttributes($dynamicGroupCategories, $catUrls, $categories);
         }
 
-        if ($this->isDirectIntegration() && !Utils::isEmpty($catUrls)) {
```

Comment:
We have one more condition for "cat_urls" in this file at line 1098

![image](https://user-images.githubusercontent.com/103488456/195824125-4bd4c948-e2c7-4afc-b683-4bed01126d90.png)

Reply:
@TobiasGraml11 if we remove this call of `isDirectIntegration()` at line 1098 line, we can remove and definition of function itself, because it is no longer used, lines 1107 - 1113.

Reply:
We need to keep `isDirectIntegration()`, in case existing (legacy) extensions still use the function.